department-of-veterans-affairs/abd-vro
# Fix presumptive flag persistence in VRO database

## What was the problem?
Presumptive flag was always false in the database

Associated tickets or Slack threads:

- [MCP-2476](https://amida.atlassian.net/browse/MCP-2476)

## How does this fix it?
Missing processing step added

## How to test this PR
Run any end to end test where presumptive flag should be true

of note:
the payload itself can have a null return from the presumptive method. That null dictates offramp reason logic. For that reason I did not change the underlying payload and have coerced the boolean when it is turned into objects where it cannot be null. only non null and true translates to database true.
